lib/kiba-common/destinations/csv.rb

```diff
@@ -0,0 +1,32 @@
+require 'csv'
+
+module Kiba
+  module Common
+    module Destinations
+      class CSV
+        attr_reader :filename, :csv_options
+        
+        def initialize(filename:, csv_options: {headers: true})
```

Comment:
Using required keyword arguments brings the benefit of getting a clear-cut error message if something is missing.